scripts/validate-artifacts.ps1

```diff
@@ -34,122 +34,93 @@ Write-Output "Validating artifacts"
 $TemplateFileText = [System.IO.File]::ReadAllText("$tempFolderPath/$resourcesTemplateName")
 $TemplateObject = ConvertFrom-Json $TemplateFileText -AsHashtable
```

Comment:
The `ConvertFrom-Json` command is used to convert the JSON content to a hashtable. Ensure that the JSON structure is correct and matches the expected format to avoid conversion errors.